docs/modules/operation/pages/deep-dive/grpc-exporter/grpc-exporter.adoc

```diff
@@ -13,17 +13,20 @@ See `monitored-services.proto` in the corresponding source distribution for the
 $ ssh -p 8101 admin@localhost
 ...
 admin@opennms()> config:edit org.opennms.features.grpc.exporter
-admin@opennms()> config:property-set host bsm.onmshs.local:1440 <1>
-admin@opennms()> config:property-set tls.cert.path /opt/opennms/etc/tls.cert <2>
-admin@opennms()> config:property-set tls.enabled false <3>
-admin@opennms()> config:property-set snapshot.interval 3600 false <4>
+admin@opennms()> config:property-set host bsm.onmshs.local:1443 <1>
```

Comment:
I think this should be:

```
admin@opennms()> config:edit org.opennms.features.grpc.exporter
admin@opennms()> config:property-set host bsm.onmshs.local <1>
admin@opennms()> config:property-set port 1443 <1>
admin@opennms()> config:property-set tenant.id opennms-prime <2>
admin@opennms()> config:property-set tls.cert.path /opt/opennms/etc/tls.cert <3>
admin@opennms()> config:property-set tls.enabled false <4>
admin@opennms()> config:property-set snapshot.interval 3600 <5>
admin@opennms()> config:update
```

Reply:
never mind - looks like host:port is ok now.

Just check re that `false` in `snapshot.interval`, wasn't sure if a 2nd argument is valid.

Reply:
good catch